modules/testsuite/cxf-tests/src/test/java/org/jboss/test/ws/jaxws/samples/wsse/policy/jaas/UsernamePasswordCallback.java

```diff
@@ -35,7 +35,7 @@ public void handle(Callback[] callbacks) throws IOException, UnsupportedCallback
    {
       WSPasswordCallback pc = (WSPasswordCallback)callbacks[0];
       if ("kermit".equals(pc.getIdentifier()))
-         pc.setPassword("thefrog");
+         pc.setPassword("therealfrog");
```

Comment:
In Elytron's realm the password is specified as "therealfrog", so it has to be changed here also

Reply:
Does it mean it gives the wrong password and expects the wrong behavior ? @Skyllarr

Reply:
@jimma The `UsernamePasswordCallback` is responsible for setting the password on client side before sending the request to server. On the server side, Elytron's realm has password "therealfrog" configured. We have to either change the password in Elytron realm or on the client side for the test to actually test the authentication. 

I just could not find where this password is configured on server in this testsuite. If we want to change the password on the server instead, you and rebeca can tell me where that is.

Reply:
@Skyllarr  I just don't understand why we only change this password without changing any test code.

Reply:
Before this PR, the test passed because of the bug where the invalid password was accepted. This bugfix has revealed that there was an invalid password from the client so I had to change it now

Reply:
@Skyllarr  Get it and thanks for the fix!